bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -34,69 +34,67 @@ const defaultProps = {
 const intlMessages = defineMessages({
   title: {
     id: 'app.presentationUploder.title',
-    defaultMessage: 'Presentation',
+    description: 'title of the modal',
   },
   message: {
     id: 'app.presentationUploder.message',
-    defaultMessage: `As a presenter in BigBlueButton, you have the ability of
-     uploading any office document or PDF file. We recommend for the best results,
-     to please upload a PDF file.`,
+    description: 'message warning the types of files accepted',
   },
   confirmLabel: {
     id: 'app.presentationUploder.confirmLabel',
-    defaultMessage: 'Start',
+    description: 'used in the button that start the upload of the new presentation',
   },
   confirmDesc: {
     id: 'app.presentationUploder.confirmDesc',
-    defaultMessage: 'Save your changes and start the presentation',
+    description: 'description of the confirm',
   },
   dismissLabel: {
     id: 'app.presentationUploder.dismissLabel',
-    defaultMessage: 'Cancel',
+    description: 'used in the button that close modal',
   },
   dismissDesc: {
     id: 'app.presentationUploder.dismissDesc',
-    defaultMessage: 'Close the modal window and discard your changes',
+    description: 'description of the dismiss',
   },
   dropzoneLabel: {
     id: 'app.presentationUploder.dropzoneLabel',
-    defaultMessage: 'Drag files here to upload',
+    description: 'message warning where drop files for upload',
   },
   browseFilesLabel: {
     id: 'app.presentationUploder.browseFilesLabel',
-    defaultMessage: 'or browse for files',
+    description: 'message use on the file browser',
   },
   fileToUpload: {
     id: 'app.presentationUploder.fileToUpload',
-    defaultMessage: 'To be uploaded...',
+    description: 'message used in the file selected for upload',
   },
   uploadProcess: {
     id: 'app.presentationUploder.upload.progress',
-    defaultMessage: 'Uploading ({progress}%)',
```

Comment:
Please confirm that the `{progress}` is not needed anymore?  - same with `{current}` `{total}`, etc

Reply:
To me it looks like it's needed - see ` progress: Math.floor(item.upload.progress).toString(),`

Reply:
I think @Tainan404 just removed the `defaultMessage` and moved to the `locale/en`

Reply:
OK! I see now! :+1: